src/ddo/DDO.ts

```diff
@@ -212,18 +272,44 @@ export class DDO {
     this.proof = await this.generateProof(publicKey)
   }
 
-  public async addService(service: any): Promise<void> {
+  /**
+   * It reorders the services of the DDO using the service index
+   */
+  public reorderServices(): void {
```

Comment:
is this just esthetics or do we rely on the DDO having the services in sorted order?

Reply:
avoid having services with duplicated indexes